client/mass/store.js

```diff
@@ -92,16 +92,25 @@ class TdbStore {
 				termfilter: JSON.parse(JSON.stringify(this.state.termfilter)),
 				termdbConfig: this.state.termdbConfig
 			})
-
+			const invalidPlots = []
 			for (const [i, savedPlot] of this.state.plots.entries()) {
 				// easier for rollup to support less complex dynamic imports with variables,
 				// webpack is already more flexible but need to support packing with rollup
 				const _ = await import(`../plots/${savedPlot.chartType}.js`)
 				const plot = await _.getPlotConfig(savedPlot, this.app)
```

Comment:
thanks i like the approach. i suggest to enclose `plot = await _.getPlotConfig()` in try-catch, so the plot code can still throw and won't be changed